test/api/api-test.js

```diff
@@ -78,6 +78,26 @@ describe('JsonApi', () => {
       })
     })
 
+    it('should allow users to add HTTP Basic Auth options', (done) => {
+      jsonApi = new JsonApi({apiUrl: 'http://myapi.com', auth: {username: 'admin', password: 'cheesecake'}})
+
+      jsonApi.define('foo', {title: ''})
+
+      let inspectorMiddleware = {
```

Comment:
I love this. I love how our tests are using our own middleware layer... so meta.